core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -931,7 +923,17 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
 
         modalInstance.result.then(function(commitMessage) {
           _modalIsOpen = false;
-          saveDraftToBackend(commitMessage);
+          saveDraftToBackend(commitMessage, function() {
```

Comment:
Isn't the backend going to check the version on which the changes are based anyway? I don't see why you need to then check this as part of the successCallback.